### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by szledan**
Patch by Attila Dusnoki and Szilard Ledan.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,3 +11,51 @@ create an engine which is:
...
+```
+./tools/scripts/install-deps.sh --developer
+```
+
+#### Our contributing process is:
+
+1. Create a branch with a name like these: `fix-min-compute`, `15-canvas2d-path`, etc.
+
+    ```
+    git checkout -b my-fisrt-patch
```
Comment (szledan) [inline README.md]: Typo: `first`.

Comment (dbatyai) [inline README.md]: Typo: first

File: README.md [hunk 2]
```diff
@@ -11,3 +11,51 @@ create an engine which is:
...
+
+## Build examples
+
+[__TODO__](https://github.com/GepardGraphics/gepard/issues/44)
+
+## For developers
+
+We use the fork model. If you want to contributing to gepard then you need to use it. ([GitHub help] (https://help.github.com/articles/working-with-forks/))
+
+After you _fork_ and _clone_ the Gepard, you will need to install more dependencies for contributing.
```
Comment (dbatyai) [inline README.md]: ... the Gepard repository, ...

File: README.md [hunk 3]
```diff
@@ -11,3 +11,51 @@ create an engine which is:
...
+./tools/scripts/build.py
+```
+
+## Build examples
+
+[__TODO__](https://github.com/GepardGraphics/gepard/issues/44)
+
+## For developers
+
+We use the fork model. If you want to contributing to gepard then you need to use it. ([GitHub help] (https://help.github.com/articles/working-with-forks/))
```
Comment (dbatyai) [inline README.md]: I'd say something like the following: 'Contribution to the project is done by using the fork model.'

File: README.md [hunk 4]
```diff
@@ -11,3 +11,51 @@ create an engine which is:
...
+
+#### Our contributing process is:
+
+1. Create a branch with a name like these: `fix-min-compute`, `15-canvas2d-path`, etc.
+
+    ```
+    git checkout -b my-fisrt-patch
+    ```
+    
+    Note: About the branch naming in Gepard. If you work on an [issue](https://github.com/GepardGraphics/gepard/issues) from Gepard then recommanded to start your branchname with the issue number.
```
Comment (dbatyai) [inline README.md]: then it is recommended to ...

**Comment by dbatyai** [comment]
lgtm

**Comment by kkristof** [comment]
lgtm

**Pull request merged**